setup.py

```diff
@@ -11,13 +11,13 @@
     history = history_file.read()
 
 requirements = [
-    'geopandas>=0.9.0,<1.0.0',
-    'pandas>1.0.0',
-    'lxml>=4.9.3',
+    'geopandas>=1.0.0,<2.0.0',
+    'pandas>2.0.0,<3.0.0',
```

Comment:
In terms of requiring pandas > 2.0, I'm worried that will break downstream users workflows who aren't ready to upgrade. Maybe that's an acceptable loss. Are the SDKs similar enough that we can leave the lower end a little more open? 
If not we will probably need to do a major bump since this is a breaking change in some ways. Not a huge deal, I just hate it when other folks make breaking changes without a proper bump 😆

Reply:
So far, things have worked w/o modification all the way up to pysnobal, awsm, firn, swiflow. So, I will give it a try with "pandas>1.0,<3.0"

Reply:
It is working for me with "pandas>1.0,<3.0"